packages/neutron-sdk/src/stargate/aux.rs

```diff
@@ -0,0 +1,28 @@
+use cosmwasm_std::{Binary, CosmosMsg, Deps, QueryRequest, StdResult};
+use prost_types::Timestamp as TimestampGen;
+use serde::de::DeserializeOwned;
+
+pub(crate) fn make_stargate_query<Req, Res>(deps: Deps, req: Req, path: &str) -> StdResult<Res>
```

Comment:
I'd add a doc comment for the func, that describes a purpose of the function, arguments and a returned value